src/devices/tuya.ts

```diff
@@ -8734,6 +8734,66 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('NAS-WV03B', ['_TZE204_rzrrjkz2']),
+        model: 'NAS-WV03B',
+        vendor: 'NEO',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        description: 'Smart Sprinkler Timer',
+        onEvent: tuya.onEventSetTime,
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            e.switch().withEndpoint('switch').withDescription('Switch'),
+            e.enum('status', ea.STATE, ['off', 'auto', 'disabled']).withDescription('Status'),
+            e.numeric('countdown', ea.STATE_SET).withUnit('min').withValueMin(1).withValueMax(240).withDescription('Countdown'),
+            e.numeric('countdown_left', ea.STATE).withUnit('min').withValueMin(1).withValueMax(240).withDescription('Countdown Left'),
```

Comment:
```suggestion
            e.numeric('countdown_left', ea.STATE).withUnit('min').withValueMin(1).withValueMax(240).withDescription('Countdown left'),
```

Reply:
Also make all the other descriptions lower case (except the first letter)

Reply:
Done